docs/source/algorithms/ResetNegatives2D-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm can be used to remove negative intensity values from workspaces created by the :ref:`Bin2DPowderDiffraction
+<algm-Bin2DPowderDiffraction>` algorithm. The algorithm looks for the lowest negative intensity value and adds this value 
+to all other intensities, leading to a workspace with 0 as the lowest intensity. The existing :ref:`ResetNegatives
+<algm-ResetNegatives>` algorithm does not produce the same results, as it looks for the lowest negative intensity in each 
+spectrum and adds this value for the corresponding spectrum only. 
+
+Usage
+-----
+
+**Example: Reset negative values from a 2D Workspace. Remember to change the Filepath for the OutputFile!**
+
+.. testcode::ResetNegatives2D
+
+    # create a 2D Workspace
+    # repeat this block for each spectrum
+    xData = [1.0,2.0,3.0,4.0,5.0,6.0,7.0,8.0,9.0]            # d values for one spectrum (one dPerpendicular value)
+    yData = ['1','2','3','4']                                # dPerpendicular binedges
+    zData = [1.0,-1.0,1.0,1.0,1.0,1.0,-2.0,1.0,1.0]          # intensity values
+    eData = [1,1,1,1,1,1,1,1,1]                              # error values
+
+    # used to join all spectra
+    xDataTotal = []                                          # d Values for all spectra
```

Comment:
I really like the idea about having some kind of output from the testcode and have implemented that already.
I am confused about what code is commented out? I cant find code that is commented out, but that might be because I usually only see python code. Maybe you can be a bit more specific about that part?

Reply:
You're right, it is functional,  I was misreading this 🤦‍♂️. But yeah, a test output would be good